runtime_config_linux.go

```diff
@@ -182,7 +182,9 @@ type CPU struct {
 
 // Pids for Linux cgroup 'pids' resource management (Linux 4.3)
 type Pids struct {
-	// Maximum number of PIDs. A value < 0 implies "no limit".
+	// Maximum number of PIDs. A value <= 0 implies "no limit".
+	// While a limit of 0 is technically valid, it is precisely identical to
+	// a limit of 1, so there is no distinction.
```

Comment:
@wking I agree (as I posted there) that #233 should in fact use pointers so that omitting and setting null have the same meaning (while also not requiring sentinel values). I'll revive this once we decide what we're going to do on #233.